runtime/onert/backend/cpu/ops/ElementwiseBinaryLayer.cc

```diff
@@ -166,6 +166,7 @@ void ElementwiseBinaryLayer::configure(const IPortableTensor *lhs, const IPortab
     case ElementwiseBinaryType::kLogicalAnd:
       if ((_lhs->data_type() == OperandType::BOOL8) && (_rhs->data_type() == OperandType::BOOL8))
       {
+        static_assert(sizeof(bool) == 1, "cpu backend supports bool type which is 1 byte");
```

Comment:
This assertion may be duplicated with the assertion in `getBuffer` below. But I'm not sure if this assertion is removed.